### PR TITLE
Rn latest safe captured time

### PR DESCRIPTION
**Pull request by Rohit25negi**
http://manage.dimagi.com/default.asp?282010

### CODE REVIEW

**Review by calellowitz — CHANGES_REQUESTED** [review]

File: custom/icds_reports/models/util.py [hunk 1]
```diff
@@ -14,6 +14,15 @@ class AggregateSQLProfile(models.Model):
     name = models.TextField()
     date = models.DateField(auto_now=True)
     duration = models.PositiveIntegerField()
+    latest_aggregation = models.DateField(null=True)
```
[the remark this answers is not in the document]
Comment (Rohit25negi) [reply custom/icds_reports/models/util.py]: 'date' field is already has that value

File: custom/icds_reports/models/util.py [hunk 2]
```diff
@@ -14,6 +14,15 @@ class AggregateSQLProfile(models.Model):
...
+
+    @classmethod
+    def get_last_indicator_acknowledged(cls):
```
[the remark this answers is not in the document]
Comment (Rohit25negi) [reply custom/icds_reports/models/util.py]: it is getting the last one only. I think the function is little misguiding. Moreover this function is not required for this change. Since fetching from this model will only required when FE will ask for it. Therefore removing this function since it is never called.

**Review by calellowitz — APPROVED** [review]
a couple small things but looks good. i do not think either is blocking (especially not the formatting one) but would prefer them

File: custom/icds_reports/models/util.py [hunk 3]
```diff
@@ -14,6 +14,7 @@ class AggregateSQLProfile(models.Model):
     name = models.TextField()
     date = models.DateField(auto_now=True)
     duration = models.PositiveIntegerField()
+    latest_aggregation = models.DateTimeField(null=True)
```
Comment (calellowitz) [inline custom/icds_reports/models/util.py]: This name is very misleading. It is not the time of the latest aggregation, but the when the most recent doc is. Can you change the name to something clearer, so others will be better able to understand this. Maybe something like `last_included_doc` or `last_included_doc_date`, or something better if you think of one.

**Review by calellowitz — CHANGES_REQUESTED** [review]

**Review by calellowitz — APPROVED** [review]

File: corehq/apps/cleanup/migrations/0012_add_es_index_to_checkpoint_ids.py [hunk 4]
```diff
@@ -60,7 +60,7 @@ class Migration(migrations.Migration):
 
     dependencies = [
         ('cleanup', '0011_merge_couch_sql_pillows'),
-        ('pillowtop', '0003_auto_20170411_1957'),
+        ('pillowtop', '0005_kafkacheckpoint_doc_modification_time'),
```
Comment (emord) [inline corehq/apps/cleanup/migrations/0012_add_es_index_to_checkpoint_ids.py]: @Rohit25negi why was this changed? I think that this is going to break on prod because this migration has already been applied. locally I saw an error like this:


```
django.db.migrations.exceptions.InconsistentMigrationHistory: Migration cleanup.0012_add_es_index_to_checkpoint_ids is applied before its dependency pillowtop.0005_kafkacheckpoint_doc_modification_time on database 'default'.
```

Comment (Rohit25negi) [reply corehq/apps/cleanup/migrations/0012_add_es_index_to_checkpoint_ids.py]: i added the because otherwise was not being included in the migration process.. hmm, i will just see how to correct this.

Comment (Rohit25negi) [reply corehq/apps/cleanup/migrations/0012_add_es_index_to_checkpoint_ids.py]: I think 0012_add_es_index_to_checkpoint_ids.py should be rerun as it depends on the pillowtop, and in this Pr i added a migration on pillowtop. so shouldn't all pillowtop's dependents needs to be rerun

Comment (Rohit25negi) [reply corehq/apps/cleanup/migrations/0012_add_es_index_to_checkpoint_ids.py]: @calellowitz
